src/heartRate.h

```diff
@@ -61,7 +61,8 @@
  #include "WProgram.h"
 #endif
 
-bool checkForBeat(int32_t sample);
+bool checkForBeat(int32_t sample, int16_t &iirFiltData, bool dcRemoved = false);
```

Comment:
Modified the function signature to have a provision to push filtered data to the calling function.

This made testing easy so i have left it in for future use.

Reply:
Added comments